doc/source/examples/extended_examples/Krylov/krylov_example.rst

```diff
@@ -308,9 +308,9 @@ The shape of the reduced solution generated is.
 
 .. code:: ipython3
 
-    result = dd.expand(residual_computation=True, residual_algorithm="l2")
+    result = dd.expand(residual_computation=True, residual_algorithm="l2",return_solution = True)
 
-Results: Pressure distribution as a function of length
+Plot the pressure distribution as a function of length
 ------------------------------------------------------
 
 Plot the pressure distribution over the length of the duct on nodes where Y, Z coordinates are zero.
```

Comment:
```suggestion
Let's plot the pressure distribution over the length of the duct on nodes where Y, Z coordinates are zero.
```

Or something to introduce better the paragraph.

Reply:
Let's is a form of Let us. Google style guidelines say not to use third person (we, they, their) but second person (you, your, yours). I'd stick with the verb form that I had.